selenium/src/test/resources/reportng-1.1.4/CHANGELOG.txt

```diff
@@ -0,0 +1,57 @@
+Changes in version 1.1.4
```

Comment:
Removed the folder reportng-1.1.4. I added it while experimenting and left it there.